src/__tests__/PojoMap.test.ts

```diff
@@ -14,4 +14,54 @@ describe('PojoMap', () => {
   it('should make an empty Map', () => {
     expect(PojoMap.empty<string, number>()).toEqual({});
   });
+
+  it('should get a value via its key', () => {
+    const map = PojoMap.fromEntries<string | number | symbol, number | boolean | string>([
+      ['a', 5],
+      [2, 'blue'],
+      ['orange', true],
+    ]);
+
+    expect(PojoMap.has(map, 'a')).toEqual(true);
+    expect(PojoMap.has(map, 2)).toEqual(true);
+    expect(PojoMap.has(map, 'orange')).toEqual(true);
+  });
+
+  it('should return has() == true even when values are falsy', () => {
+    const map = PojoMap.fromEntries<string, number | boolean | string>([
+      ['a', 0],
+      ['b', false],
+      ['c', ''],
+    ]);
+    expect(PojoMap.has(map, 'a')).toEqual(true);
+    expect(PojoMap.has(map, 'b')).toEqual(true);
+    expect(PojoMap.has(map, 'c')).toEqual(true);
+    });
+
+    it('should add to a PojoMap immutably', () => {
+      const map = PojoMap.fromEntries<string, number>([
+        ['a', 0],
+        ['b', 1],
+        ['c', 2],
+      ]);
+      const largerMap = PojoMap.set(map, 'd', 3);
+      expect(largerMap).toEqual({
+        a: 0,
+        b: 1,
+        c: 2,
+        d: 3,
+      });
+    });
+
+    it('should remove from a PojoMap immutably', () => {
```

Comment:
Can you add a test that tries to remove a key from the map even when the key is not there?

e.g. `PojoMap.remove(map, 'c')`

(This just just return a copy of the input map)